modules/optionsModal.js

```diff
@@ -62,9 +61,9 @@ openModalButton.addEventListener('click', showModal);
 // When the user clicks on <span> (x), close the modal
 closeModalButton.addEventListener('click', hideOptionsModal);
 
-autoSort.addEventListener('change', populateItems);
-hideChecked.addEventListener('change', populateItems);
-sectionSort.addEventListener('change', populateItems);
+autoSort.addEventListener('change', window.populateItems);
+hideChecked.addEventListener('change', window.populateItems);
+sectionSort.addEventListener('change', window.populateItems);
```

Comment:
I bound this fn to the `window` in `index.ts` - but this might not be the best way to do this 🤔